src/main/java/algorithms/curated170/medium/SortTransformedArray.java

```diff
@@ -0,0 +1,47 @@
+package algorithms.curated170.medium;
+
+public class SortTransformedArray {
+
+        int n, start = 0, end;
+        int[] res;
+        int[] nums;
+        int a, b, c;
+
+        public int[] sortTransformedArray(int[] nums, int a, int b, int c) {
+            n = nums.length;
+            res = new int[n];
+            end = n-1;
+            this.nums = nums;
+            this.a = a;
+            this.b = b;
+            this.c = c;
+            
+            if (a > 0) {
+                for (int i = n - 1; i >= 0; i--) {
+                    quarticComparator(i, true);
```

Comment:
This should be called quadraticComparator, quartic is something relating to the power of 4 (x^4).